keyboards/kinesis/keymaps/kzar/keymap.c

```diff
@@ -0,0 +1,398 @@
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+enum layer_names {_QWERTY, _DVORAK, _MAC, _PC, _KEYPAD, _PROGRAM};
+enum my_keycodes {QWERTY = SAFE_RANGE, DVORAK, WIN, MAC, PC, STATUS, PROGRAM};
+
+#define ledTurnOff writePinHigh
+#define ledTurnOn writePinLow
+
+#define LED_KEYPAD LINE_PIN24
```

Comment:
This line is causing Travis to fail, do you have any idea what I should use instead of `LINE_PIN24` now?

Reply:
Hmm `E26` seems to work for me, but Travis complains it's undefined too. Weird.

Reply:
The travis failure seems to originate from `kinesis/alvicstep with keymap kzar`.

I suppose using the `E26` pin makes your keymap specific to the kint36 keyboard, and incompatible with e.g. the alvicstep keyboard.

Reply:
Done.